patches/tools-perf-core-perfetto_binary_roller-binary_deps_manager.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tools/perf/core/perfetto_binary_roller/binary_deps_manager.py b/tools/perf/core/perfetto_binary_roller/binary_deps_manager.py
+index 1619dcb78a5262e9de6cefa6644012281eca136c..6780ace5640c0bcf2d4c22e320e1d2eb45b4c384 100644
+--- a/tools/perf/core/perfetto_binary_roller/binary_deps_manager.py
++++ b/tools/perf/core/perfetto_binary_roller/binary_deps_manager.py
+@@ -229,6 +229,9 @@ def FetchHostBinary(binary_name):
+   expected_hash = config[binary_name][platform]['hash']
+   filename = posixpath.basename(remote_path)
+   local_path = os.path.join(LOCAL_STORAGE_FOLDER, filename)
++  if os.path.exists(local_path) and cloud_storage.CalculateHash(local_path) == expected_hash:
```

Comment:
Trying to upstream this change in https://chromium-review.googlesource.com/c/chromium/src/+/6795468/1